crates/ty_python_semantic/resources/mdtest/final.md

```diff
@@ -29,3 +29,344 @@ class H(
     G,
 ): ...
 ```
+
+## Cannot override a method decorated with `@final`
+
+<!-- snapshot-diagnostics -->
+
+```pyi
+from typing_extensions import final, Callable, TypeVar
+
+def lossy_decorator(fn: Callable) -> Callable: ...
+
+class Parent:
+    @final
+    def foo(self): ...
+    @final
+    @property
+    def my_property1(self) -> int: ...
+    @property
+    @final
+    def my_property2(self) -> int: ...
+    @final
+    @classmethod
+    def class_method1(cls) -> int: ...
+    @staticmethod
+    @final
+    def static_method1() -> int: ...
+    @final
+    @classmethod
+    def class_method2(cls) -> int: ...
```

Comment:
What's the difference between `class_method1` and `2` tha makes it worth having both?

The same for `static_method1` and `2`. Did you mean to change the decorator ordering?